Packages/SmithyTestUtil/Sources/XCTAssertions.swift

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import XCTest
+
+public func XCTAssertJSONDataEqual(
+    _ expression1: @autoclosure () throws -> Data,
+    _ expression2: @autoclosure () throws -> Data,
+    _ message: @autoclosure () -> String = "",
+    file: StaticString = #filePath,
+    line: UInt = #line
```

Comment:
Hmmm, never thought of using `#filePath` and `#line` in test helpers & custom assertions (only ever used it in logging statements.)

Reply:
Yeah its nice to use here so that the assertion failure will create a nice error message at the proper line in the test. Otherwise, it can be hard to figure out the cause of the error.